src/text-field-base.tsx

```diff
@@ -358,6 +359,21 @@ const TextFieldBaseComponent = React.forwardRef<any, TextFieldBaseProps>(
             labelStyle = {paddingRight: 36};
         }
 
+        // render icons of disabled fields with neutralLow color
+        const renderIcon = (icon: React.ReactNode) => {
+            if (!rest.disabled) {
+                return icon;
+            }
+            if (!isElement(icon)) {
+                return icon;
+            }
+            if (icon.props.color) {
+                // if a explicit color is set, use it
+                return icon;
+            }
```

Comment:
Could it be useful to reduce the opacity or something like that in the previous two cases? in order to let the textfield to control the disabled state even when the icon is not an icon:
![image](https://user-images.githubusercontent.com/681159/133977022-0ef346e3-ccdf-4ffa-a822-1916cbd76278.png)

Reply:
I think this could be the best approach for all cases. If disabled, set an opacity to the icon container. No need to define different color for enabled/disabled states

Reply:
I like this approach too. I talked with Abel moooonths ago about to have a constant style to disabled things.

Could be a good opportunity to have a "disabled" state with opacity for any thing disabled in Mística.

WDYT @atabel ?

Reply:
Do you think that'd work in all the cases @yceballost? including all brands and dark mode?

Reply:
well, exists some cases like buttons disabled that have their own style. Maybe we can try if buttons works with this definition too and remove disabled buttons constant too.

Reply:
applied opacity instead of different color constant. But only for textfield icons for the moment, please take a look